gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-documentation/api-tab-documentation.component.scss

```diff
@@ -30,6 +30,9 @@
 
   &__page-content {
     width: 100%;
+    max-width: calc(100% - 276px - 36px);
```

Comment:
Is the max-width necessary if you use `break-word`; ?
I would like to avoid magic calculation of the page width as much as possible